app/views/layouts/rails_admin/_sidebar_navigation.html.erb

```diff
@@ -1,9 +1,5 @@
-<ul class="nav nav-pills nav-stacked">
+<ul class="col-sm-3 col-md-2 btn-toggle-nav list-unstyled bg-light">
```

Comment:
I struggled a lot with this change because `position: fixed` is used here, but finally got equivalent appearance.